docs/architecture/009-review-and-form-model.md

```diff
@@ -0,0 +1,93 @@
+
+All in all while multiple forms, and **option 3** are a sensible extension to our model, it certainly requires large changes to the logic all around the codebase.
+
+In the **option 4** we would make a new repeating functionality that would repeat per applied catalogue item. Modeling each reviewer organization as a catalogue item seems like a business decision, not something for us. It requires the organization to either be able to statically know each option, or have a process for adding missing ones. While the repeating logic is sound, we can also consider **option 5** as the same, but not having the need to couple this into catalogue items. So lets prefer **option 5** and not this one.
+
+A field type where you can "press add button" to add more rows sounds also like a natural extension to the types of fields we have. Also repeating fields based on the count of a previous answer sounds sensible. Some kind of grouping logic may be needed to repeat fields in a group (ABCABCABC and not AAABBBCCC). Therefore **option 5** seems like a sensible extension that we should consider.
+
+Review request must in practice be changed in all of these. This is due to the fact that receiving organizations can't easily be modeled, they can be email inboxes. Therefore there should be an option to select the organization type i.e. which private fields are shown to them. This would be part of all the solutions, either picking from a catalogue item, resource, organization or answer to a multiple answer field.
+
+It is likely that reviewers don't have existing REMS user accounts, and certainly it's not possible to select which one should do the review, because of the email group box requirement. Therefore inviting a reviewer to review, and connecting the person to the application and/or organization, must be done like in the invite member function.
+
+Last but not least there is the **option 6**. So what if we want to model the registry keeping organizations as REMS organizations and catalogue items, and/or registries as catalogue items? We could have review requests targeted to organizations, where organization would define the email inboxes and field visibility. A reviewer would see all public form fields, and also private fields of the organization they represent. Common fields can be put as workflow form fields, if we add the capability. This is close to what we do with licenses at the moment, one can add them to resources as well as workflows.
+
+## Decision
+
+Let's try **option 6** because we anticipate that there is a need for multiple forms, as well as a wish to model the organizations better inside REMS.
+
+The tasks should be
+1. organization creation
+   - store id, name, description, owners and review-email
```

Comment:
I'm not sure this concept for organizations is compatible with the current concept for organizations. First of all, currently we assume users are mapped to organizations by the idp, but in this case the mapping would be dynamic. Perhaps there wouldn't even be a need to map users to organizations since the review request constains the organization?

Could there be two different concepts for organization? This new one could be called a "visibility group" or something.

Reply:
Here are some considerations:
- A review request can contain an organization but it may not contain a user. It is unknown by the handler who the reviewer will be. 
- Not all users logging into an organization should be able to see all reviews.
- Does IDP organization for a reviewer actually match expected organization? I think the mapping to a specific application must be done like the invite member.
- I don't see any problem in having one concept for an organization here and using it for two things. For example we can avoid having two separate but almost identical UI functions. 
- The organization that does reviews also likely wants to maintain their own forms and/or catalogue, which is exactly the organization model we have now?

Reply:
I think the leftover from this discussion was just that we slightly change organization concept and check with other users that there is no conflict there.